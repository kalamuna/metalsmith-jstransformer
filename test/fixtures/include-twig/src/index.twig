---
buttons:
  - Hello
  - World
  - How
  - Are
  - You
---
<h1>Buttons</h1>
<div class="buttons">
  {% include 'components/button-group/_button-group.twig' with buttons only %}
</div>
