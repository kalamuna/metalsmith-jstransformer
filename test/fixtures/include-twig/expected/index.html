<!DOCTYPE html>
<html>
  <head>
    <title>My Site</title>
  </head>
  <body>This is my site!
</body>
</html>